.github/workflows/docker-image.yml

```
@@ -67,6 +67,10 @@ jobs:
         id: build
         uses: docker/build-push-action@v6
         with:
+          build-args: |
```

actionlint (1.7.7)

property "revision" is not defined in object type {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}

(expression)